webpack.config.js

```diff
@@ -2,6 +2,7 @@
  **** WARNING: No ES6 modules here. Not transpiled! ****
  */
 /* eslint-disable import/no-nodejs-modules */
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
what was this tripping?

Reply:
I had was getting errors for `duplicate-package-checker-webpack-plugin`, `webpack`, but I've backed this change out for now.

```
  11:17  error  'webpack' should be listed in the project's dependencies. Run 'npm i -S webpack' to add it                                                                    import/no-extraneous-dependencies
  15:39  error  'duplicate-package-checker-webpack-plugin' should be listed in the project's dependencies. Run 'npm i -S duplicate-package-checker-webpack-plugin' to add it  import/no-extraneous-dependencies
```

Reply:
ah, those are real errors. we're getting away with not having them in the root package.json because of how npm is current hoisting dependencies from `packages/calypso-build`. If we `require` or `import` them, they should be in the closest `package.json`.